packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -110,7 +124,34 @@ class ExportToCsvCommand extends Command {
             }
 
             stackAPIClient = this.getStackClient(managementAPIClient, stack);
+
+            if (branchUid) {
+              try {
+                const branchExists = await doesBranchExist(stackAPIClient, branchUid);
+                if (branchExists && branchExists.errorCode) {
+                  throw new Error(branchExists.errorMessage);
+                }
+                stack.branch_uid = branchUid;
+                stackAPIClient = this.getStackClient(managementAPIClient, stack);
+              } catch (error) {
+                if (error.message || error.errorMessage) {
```

Comment:
There is a Sonar-lint issue on this line.

Reply:
fixed it